### PR TITLE
Filter products by course seat type

### PR DESCRIPTION
**Pull request by marjev**

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=943&repo=edx%2Fecommerce

**Comment by marjev** [comment]
@mjfrey Here's the PR that fixes the problem with Offer Landing Page (not filtering the course seats by seat type).

**Comment by mjfrey** [comment]
👍

File: ecommerce/extensions/api/v2/views/vouchers.py [hunk 1]
```diff
@@ -115,9 +115,13 @@ def get_offers(self, request, voucher):
...
             courses = Course.objects.filter(id__in=course_ids)
-            products = Product.objects.filter(course__in=courses)
+            products = Product.objects.filter(
+                attributes__name='certificate_type',
+                attribute_values__value_text__in=course_seat_types.split(','),
+                course__in=courses
+            )
             stock_records = StockRecord.objects.filter(product__in=products)
-            contains_verified_course = (benefit.range.course_seat_types == 'verified')
+            contains_verified_course = (course_seat_types == 'verified')
```
Comment (vkaracic) [inline ecommerce/extensions/api/v2/views/vouchers.py]: Why are brackets needed here?

**Review by vkaracic — APPROVED** [review]
Small nit question, otherwise :+1:

**Pull request merged**